### DISCUSSION TITLE
Define simulation from the main menu

### DISCUSSION THREAD

**Discussion by ra3xdh** [Caneda/Caneda#27] [Ideas]
Currently Caneda uses the "everything is component" concept. The simulations are also the components on schematic. The usage of Qucs and Qucs-S has shown that the users don't understand this concept. The most of users find an interface like in LTSpice more user-friendly. The Caneda already has the waveform viewer that is more user-friendly.

I would advise to add the possibility to define the simulations from the main menu. Caneda is in early development stage and it is not too late to make a redesign in the GUI.  The menu points like *Simulation->Transisent* or *Simulation->AC* could be added.

**Comment by parejaobregon** [comment]
@ra3xdh I like your suggestion. Currently I have several items in my TODO, however I don't think it should be that hard to implement this. Perhaps as an invisible simulation component that could co-exist with the current implementation?

**Comment by ra3xdh** [comment]
Yes, the insertion of the invisible simulation may be a solution.
